packages/codec/tests/bytes-like.test.ts

```diff
@@ -1,7 +1,7 @@
 import test from "ava";
 import { molecule, number } from "../src";
 import { Byte32 } from "../src/blockchain";
-import { randomBytes } from "crypto";
+import { randomBytes } from "../../crypto/src";
```

Comment:
Using the workspace feature offered by Pnpm to avoid the uninstalled package error. 

```suggestion
import { randomBytes } from "@ckb-lumos/crypto";
```

Reply:
Updated. 
Also added `"@ckb-lumos/crypto"` to `devDependecies` in `package.json` to avoid red wavy line in VSCode/IDE.